api/src/main/java/io/opentelemetry/metrics/DefaultMeter.java

```diff
@@ -164,6 +165,12 @@ public LabelSet createLabelSet(
     return NoopLabelSet.INSTANCE;
   }
 
+  @Override
+  public LabelSet createLabelSet(@Nonnull Map<String, String> labels) {
```

Comment:
We don't use Nonnull anywhere else, probably just remove it for the moment.

Reply:
done